frontend/src/css/modules/base.scss

```diff
@@ -82,13 +82,19 @@ a {
 ::-webkit-scrollbar {
```

Comment:
@aayusharora Please explain why writing it in a single css line with a comma is better ?